src/doc/unstable-book/src/compiler-flags/patchable-function-entry.md

```diff
@@ -0,0 +1,24 @@
+# `patchable-function-entry`
+
+--------------------
+
+The `-Z patchable-function-entry=M,N` or `-Z patchable-function-entry=M`
+compiler flag enables nop padding of function entries with M nops, with
+an offset for the entry of the function at N nops. In the second form,
+N defaults to 0.
```

Comment:
This should mention the "total,prefix" mnemonic somewhere.

Reply:
I replaced M and N with total_nops and prefix_nops. Although it now sounds a bit clunky, it should be more consistent and clear.